packages/components/src/Button/Button.css

```diff
@@ -30,16 +30,27 @@
   justify-content: center;
 }
 
+:global(.jobber-retheme) .button {
+  min-height: calc(var(--base-unit) * 2.5);
+}
+
 .button * {
   color: inherit;
   fill: inherit !important;
   /* required to over-ride very specific fill from <Icon> */
 }
 
+:global(.jobber-retheme) .button * {
+  font-weight: 600 !important;
+  line-height: 1;
+  letter-spacing: none;
```

Comment:
i believe this has to be a unit and be 0

Reply:
Say no more!